client/src/game/types/ReplayState.ts

```diff
@@ -1,9 +1,19 @@
+import CardIdentity from './CardIdentity';
 import GameState from './GameState';
 
 export default interface ReplayState {
   readonly active: boolean;
   readonly turn: number;
   readonly states: readonly GameState[];
-  readonly ongoingHypothetical: GameState | null;
-  readonly hypotheticalStates: readonly GameState[];
+  readonly hypothetical: HypotheticalState | null;
+}
+
+export interface HypotheticalState {
```

Comment:
doesn't this belong in its own file

Reply:
Potentially. one could argue we can make a hypo reducer as well... I'm going for parity first :)